stanity/psisloo.py

```diff
@@ -0,0 +1,78 @@
+
+import psis
+import pandas
+import math
+import numpy
+import seaborn
+import matplotlib as pyplot
```

Comment:
looks like this is unused anyway so should be dropped, but fwiw pyplot is a module within matplotlib, so you'd want `from matplotlib import pyplot`